study-builder/studies/pancan/subs.conf

```diff
@@ -139,8 +139,16 @@
       && user.studies["cmi-pancan"].forms["PREQUAL"].questions["PRIMARY_CANCER_LIST_SELF"].children["PRIMARY_CANCER_SELF"].answers.hasAnyOption("C_GASTRO_ESOPHAGEAL_CANCER", "C_GASTRO_GASTRIC_STOMACH_CANCER"))
     """,
     "has_lms": """
-      (user.studies["cmi-pancan"].forms["PREQUAL"].questions["PRIMARY_CANCER_LIST_SELF"].children["PRIMARY_CANCER_SELF"].answers.hasOptionStartsWith("C_SARCOMAS_S_LEIOMYO")
-      || user.studies["cmi-pancan"].forms["PREQUAL"].questions["PRIMARY_CANCER_LIST_CHILD"].children["PRIMARY_CANCER_CHILD"].answers.hasOptionStartsWith("C_SARCOMAS_S_LEIOMYO"))
+      (
+        (user.studies["cmi-pancan"].forms["PREQUAL"].questions["PRIMARY_CANCER_LIST_SELF"].children["PRIMARY_CANCER_SELF"].answers.hasOptionStartsWith("C_SARCOMAS_S_LEIOMYO")
```

Comment:
If its just 2 or 3 stableIds and they start with different string pattern like here we can use "hasAnyOption" rather than 3 pex with a  ||
For ex: here we can use:
(user.studies["cmi-pancan"].forms["PREQUAL"].questions["PRIMARY_CANCER_LIST_SELF"].children["PRIMARY_CANCER_SELF"].answers.hasAnyOption("C_GYNECOLOGIC_UTERINE_LEIOMYOSARCOMA", "C_SARCOMAS_S_LEIOMYO", "C_SARCOMA_CUTANEOUS_LEIMYOSARCOMA")

Reply:
Thank you- that's helpful to know!